shipyard-controller/handler/shipyardcontroller.go

```diff
@@ -150,33 +151,40 @@ func (sc *shipyardController) HandleIncomingEvent(event models.Event, waitForCom
 			if err != nil {
 				log.WithError(err).Error("Unable to handle sequence '.triggered' event")
 			}
-			complete(err)
-		}()
-	case string(common.StartedEvent):
-		go func() {
-			err := sc.handleTaskStarted(event)
-			if err != nil {
-				log.WithError(err).Error("Unable to handle task '.started' event")
-			}
-			complete(err)
+			cb(err)
 		}()
-	case string(common.FinishedEvent):
+	case string(common.StartedEvent), string(common.FinishedEvent):
 		go func() {
-			err := sc.handleTaskFinished(event)
+			err := sc.handleTaskEvent(event)
 			if err != nil {
-				log.WithError(err).Error("Unable to handle task '.finished' event")
+				if errors.Is(err, ErrSequenceNotFound) || errors.Is(err, models.ErrInvalidEventScope) {
```

Comment:
events with invalid scopes should not be logged with an error log level - this should reduce log pollution